src/app/public/modules/dropdown/dropdown-extensions.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  SkyAffixHorizontalAlignment
+} from '@skyux/core';
+
+import {
+  SkyDropdownHorizontalAlignment
+} from './types/dropdown-horizontal-alignment';
+
+export function parseAffixHorizontalAlignment(
```

Comment:
I want to check that specific string values are represented by the dropdown's specific type. It should throw an error if a new type is created by the affix module, but not represented by the dropdown so that we can make the necessary changes without affecting the consumer.

Reply:
I had to do something similar for the C# notifications service and thought it would apply similarly here.

Reply:
Seems prudent enough. 👍